example-react-app/src/app/custom-data-display-components/CustomDataDisplayComponents.tsx

```diff
@@ -1,52 +1,60 @@
 import {registerScreen} from "@haulmont/jmix-react-web";
-import styles from "../App.module.css";
-import {Card} from "antd";
+import {Card, Space} from "antd";
 import { EntityHierarchyTree } from "@haulmont/jmix-react-antd";
+import { Label } from "@haulmont/jmix-react-web"
 
-const ROUTING_PATH = "/CustomDataDisplayComponents";
+const ROUTING_PATH = "/customDataDisplayComponents";
 
 const CustomDataDisplayComponents = () => {
-  return (
-    <Card className={styles.narrowLayout} title="EntityHierarchyTree">
-      <EntityHierarchyTree
-          hierarchyProperty="parent"
-          items={[
-            {
-              id: "A",
-              _instanceName: "Node A",
-          }, {
-              id: "A1",
-              parent: "A",
-              _instanceName: "Node A1",
-          }, {
-              id: "A2",
-              parent: "A",
-              _instanceName: "Node A2",
-          }, {
-              id: "A21",
-              parent: "A2",
-              _instanceName: "Node A21",
-          }, {
-              id: "B",
-              _instanceName: "Node B",
-          }, {
-              id: "B1",
-              parent: "B",
-              _instanceName: "Node B1",
-          }]}
-      />
-    </Card>
-  )
+    return (
+        <Space align="start" wrap>
+            <Card title="EntityHierarchyTree">
```

Comment:
Let's save layout like in other screens:
```suggestion
        <Card className={styles.narrowLayout}>
            <Space direction="vertical">
```
![49403d4c8aea724b65b62447eabfd5d4](https://user-images.githubusercontent.com/34520941/143562130-10140e66-b64b-4d93-b313-a664b5251631.png)
You can add labels for the components if you think it's needed